File: Programas em C/04 - Laço For com Vetores/script3.c
/*
 * Faça um programa que carregue um vetor com dez numeros
 * inteiros. Mostre o vetor na ordem inversa a que foi digitado.
 */

#include <stdio.h>

int main(){
	//variaveis
	int vetor[10];//0...9
	//entradas
	for(int i = 0; i < 10; i++){
		printf("Informe o valor para o vetor: ");
		scanf("%d", &vetor[i]);
	}
	for(int i = 9; i >= 0; i--){
		printf("%d\n", vetor[i]);
	}

}

File: Programas em Python/04 - Vetores/Script3.py
# Faça um programa que carregue um vetor com dez numeros 
# inteiros. Mostre o vetor na ordem inversa a que foi 
# digitado.

#variaveis
vetor = []
#entradas
for n in range(0, 10):
    num = int(input("Informe o valor para o vetor: "))
    vetor.append(num)
#processamento
vetor.reverse() #inverte a lista
for n in vetor:
    print(n)
